activity_log.php

```diff
@@ -30,9 +30,6 @@
 $sys = ( $is_admin && getGetValue ( 'system' ) != '' );
 
 print_header();
-
-ob_start();
```

Comment:
I meant, I screwed up the master on my github copy.
git commit and push there.
I tried cloning from your original and my fork. I couldn't figure out how to change "yours" to mine on my computer. And mine is toast.
I imagine that github newbies do this a lot. It's really easy to mess it up. There must be a way to undo or re-sync. At least, I hope there is.